### ISSUE TITLE
Images in older articles don't have an option to open images in the image viewer/fullscreen

### ISSUE THREAD

**Issue by jlg97** [guardian/frontend#19768]
You can't open images/photos in the image viewer on older articles.

For example in these articles below:

https://www.theguardian.com/travel/2011/aug/17/10-best-clubs-berlin
https://www.theguardian.com/law/2010/sep/27/government-torture-guidance
https://www.theguardian.com/global/2010/nov/11/4d-man-modern-masculinity

**Comment by paperboyo** [comment]
Yeah, the site prevents Lightbox on low resolution images like those above – there wouldn’t be much to gain there, although, maybe, one could argue that it would still offer useful functionality on mobile. And it would be consistent.

What I find more objectionable is for the site to also block certain ratio images irregardless of the resolution.

For example in these articles below:

https://www.theguardian.com/artanddesign/2018/apr/15/surface-work-victoria-miro-review-women-abstract-painting
https://www.theguardian.com/culture/picture/2017/dec/02/modern-toss-cartoon-loo-of-the-year-awards

While I understand the intentions here were noble, the effect is I have to scroll around. I hate scrolling around.

We should probably get rid of both restrictions. If not both – just the second one.

**Comment by paperboyo** [comment]
Also, related to first issue: https://github.com/guardian/frontend/issues/12215.
